hera_cal/lstbin_simple.py

```diff
@@ -0,0 +1,1677 @@
+"""
+An attempt at a simpler LST binner that makes more assumptions but runs faster.
+
+In particular, we assume that all baselines have the same time array and frequency array,
+and that each is present throughout the data array. This allows a vectorization.
```

Comment:
Can you add, 

```
The main binning function is `lst_bin_files_for_baselines` which has a similar but not identical API as `lstbin.lstbin`
```

Reply:
I actually think the function that retains the most similarity to the old lstbin module is the `lst_bin_files`, which is different in the new code because it receives a pre-configured "config" file specifying the files to use (and the output LST grid), but otherwise serves the same function (i.e. takes in files, outputs files). There is actually nothing quite similar to the old `lstbin.lst_bin`, which took in a list of data *objects* and returned a list of LST-binned data and their averages. To acheive that in the new code, you would call `lst_align` followed by `reduce_lst_bins`. In contrast, `lst_bin_files_for_baselines` also takes as input the data *files*, and outputs non-averaged lst-bins only. 

I've added some comments in the module docstring to make the point that the main entry-point is the `lst_bin_files` function (and that it is similar to the function of the same name in the old module). I could add more docs about how to use the module as a whole, if you like. I could also add a function that is more similar to the old `lst_bin()`, which would be a small wrapper around the existing functions. It wouldn't be super-duper easy, though not too hard.

Reply:
I actually think the function that retains the most similarity to the old lstbin module is the `lst_bin_files`, which is different in the new code because it receives a pre-configured "config" file specifying the files to use (and the output LST grid), but otherwise serves the same function (i.e. takes in files, outputs files). There is actually nothing quite similar to the old `lstbin.lst_bin`, which took in a list of data *objects* and returned a list of LST-binned data and their averages. To acheive that in the new code, you would call `lst_align` followed by `reduce_lst_bins`. In contrast, `lst_bin_files_for_baselines` also takes as input the data *files*, and outputs non-averaged lst-bins only. 

I've added some comments in the module docstring to make the point that the main entry-point is the `lst_bin_files` function (and that it is similar to the function of the same name in the old module). I could add more docs about how to use the module as a whole, if you like. I could also add a function that is more similar to the old `lst_bin()`, which would be a small wrapper around the existing functions. It wouldn't be super-duper easy, though not too hard.

Reply:
cool thanks for the added comments